addon/index.js

```diff
@@ -33,7 +32,7 @@ let Notify = Service.extend({
       text = null;
     }
 
-    let message = Message.create(assign({
+    let message = Message.create(Object.assign({
       text: text,
       type: type
     }, options));
```

Comment:
This works, although can also just:
```js
let message = Message.create({
  text,
  type,
  ...options
});
```

Reply:
I'm good with either - I was just using the most minimal change to fix the deprecation to unblock consumers. If maintainers want to edit to that version instead, that's fine with me.